### PR TITLE
Simplify merton

### PR DESCRIPTION
**Pull request by MirijaHa**

### CODE REVIEW

File: R/calc_survival_probability_merton.R [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+#' Function calculates survival probability for a maturity based on a structural
+#' Merton model.
+#' For details on implementation please compare [CreditRisk::Merton()].
+#' Unlike [CreditRisk::Merton()] this implementation:
+#' 1. only holds functionality to calculate probability of survival
+#' 1. can be called in vectorised fashion
+#' 1. has additional input validation by checking that all input values are
+#' non negative numeric vectors of the same length.
+#'
+#' @param L Numeric vector, holding dept values at maturity.
```
Comment (jacobvjk) [inline R/calc_survival_probability_merton.R:12]: "debt"

File: R/calc_survival_probability_merton.R [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+#' @param V0 Numeric vector, holding values at time t0.
+#' @param sigma Numeric vector, holding volatility values.
+#' @param r Numeric vector, holding risk free rates.
```
Comment (jacobvjk) [inline R/calc_survival_probability_merton.R:15]: risk free interest rate

File: R/calc_survival_probability_merton.R [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+#' Merton model.
+#' For details on implementation please compare [CreditRisk::Merton()].
+#' Unlike [CreditRisk::Merton()] this implementation:
+#' 1. only holds functionality to calculate probability of survival
+#' 1. can be called in vectorised fashion
+#' 1. has additional input validation by checking that all input values are
+#' non negative numeric vectors of the same length.
+#'
+#' @param L Numeric vector, holding dept values at maturity.
+#' @param V0 Numeric vector, holding values at time t0.
```
Comment (jacobvjk) [inline R/calc_survival_probability_merton.R:13]: company values

File: R/calc_survival_probability_merton.R (hunk 2 above)
Comment (MirijaHa) [reply R/calc_survival_probability_merton.R:15]: just to make sure, in the original description it says risk free rate without interest, are those two redunant?

**Pull request merged**